docs/providers/aws/guide/serverless.yml.md

```diff
@@ -928,6 +928,32 @@ functions:
           enabled: false
 ```
 
+### ActiveMQ
+
+[ActiveMQ events](../events/activemq.md):
+
+```yaml
+functions:
+  hello:
+    # ...
+    events:
+      - activemq:
+          # ARN of ActiveMQ Broker
+          arn: arn:aws:mq:us-east-1:0000:broker:ExampleMQBroker:b-xxx-xxx
+          # name of ActiveMQ queue consume from
+          queue: queue-name
+          # Secrets Manager ARN for credentials required to do basic auth
```

Comment:
let's write it as `Secrets Manager ARN for basic auth credentials`

Reply:
Fixed in 952ae3e404ad26c6331dd3f882ea7137efaeb031 #